docs/tutorials/intro_tutorial.rst

```diff
@@ -879,273 +895,208 @@ True indefinitely.
                 x = self.random.randrange(self.grid.width)
                 y = self.random.randrange(self.grid.height)
                 self.grid.place_agent(a, (x, y))
-
+    
             self.datacollector = DataCollector(
-                model_reporters={"Gini": compute_gini},
-                agent_reporters={"Wealth": "wealth"})
-
+                model_reporters={"Gini": compute_gini}, agent_reporters={"Wealth": "wealth"}
+            )
+    
         def step(self):
             self.datacollector.collect(self)
             self.schedule.step()
 
-We instantiate a BatchRunner with a model class to run, and two
-dictionaries: one of the fixed parameters (mapping model arguments to
-values) and one of varying parameters (mapping each parameter name to a
-sequence of values for it to take). The BatchRunner also takes an
-argument for how many model instantiations to create and run at each
-combination of parameter values, and how many steps to run each
-instantiation for. Finally, like the DataCollector, it takes
-dictionaries of model- and agent-level reporters to collect. Unlike the
-DataCollector, it won't collect the data every step of the model, but
-only at the end of each run.
+We call ``batch_run`` with a model class to run, and a dictionary with
+the model parameters where each of the values can either be fixed or an
+iterable. Further, we can specify how many model instantiations to
+create and run at each combination of parameter values, and how many
+steps to run each instantiation for. We can also set the number of
+processors to be run at the same time (default ``None`` uses all
```

Comment:
You haven't said that those processors are used to run the sweep in parallel.

Reply:
Yes, let's highlight the multiprocessing point as multiprocessing can often get temperamental depending on the OS. 

Again a thought, but feel free to not use or make better:

_Further, we can specify how many model instantiations to create and run at each combination of parameter values, and how many steps to run in each instantiation. By default, ```batch_run``` will use every processor so the different model variations are run in parallel. Multiprocessing does make debugging challenging. If your parameter sweeps are resulting in unexpected errors set ```nr_processes = 1```.  ```batch_run``` can also set the number of steps that data is collected with with keyword argument  ```data_collection_period```._